### PR TITLE
Use datatype from model

### PR DESCRIPTION
**Pull request by bastbijl**
Read default data type for attributes fro model.

I certify that:
- [x] The automated build passes on the branch this pull request is from
- [x] My change is documented in the Changelist.md
- [x] Any functionality additions/changes are documented in the README.md
- [x] Any functionality additions/changes are _also_ documented [here](https://github.com/weaverplatform/weaver-docs/blob/master/pages/developers/reference/weaver-sdk-js.md "Weaver-docs")
- [x] Any new dockerimages this build depends on are properly tagged on github
- [x] I shall not merge until the travis build is green and the PR is accepted by
  another developer
- [x] I shall promptly fix/reply to comments, as I know PRs are not :fire: and
  forget
- [x] I shall keep my branch updated with any intermediate changes to the
  target branch
- [x] This is a quality PR
- [x] I got to the end of this list

### CODE REVIEW

File: Changelist.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelist
 
+## Develop
+- Read default data type for attributes fro model.
```
Comment (gvdent) [inline Changelist.md]: from* ;)

File: src/WeaverModelClass.coffee [hunk 2]
```diff
@@ -61,10 +61,19 @@ class WeaverModelClass extends Weaver.Node
 
     if not @totalClassDefinition.attributes?
       throw new Error("For #{@id()} of class #{@className} model is not allowed to have attributes")
-    if not @totalClassDefinition.attributes[field]?
+    if field not in Object.keys(@totalClassDefinition.attributes)
```
Comment (gvdent) [inline src/WeaverModelClass.coffee]: Why this change? Afaik the functionality is exactly the same but getting an array of keys and doing the lookup in that is both slower and imo looks worse

File: src/WeaverModelClass.coffee [hunk 3]
```diff
@@ -61,10 +61,19 @@ class WeaverModelClass extends Weaver.Node
...
 
-    @totalClassDefinition.attributes[field].key or field
+    @totalClassDefinition.attributes[field]?.key or field
+
+  _getAttributeKeyDataType: (field) ->
+
+    if not @totalClassDefinition.attributes?
+      throw new Error("#{@className} model is not allowed to have attributes")
+    if field not in Object.keys(@totalClassDefinition.attributes)
+      throw new Error("#{@className} model is not allowed to have the #{field} attribute")
```
Comment (gvdent) [inline src/WeaverModelClass.coffee]: These two checks can be combined to `if not @totalClassDefinition.attributes?[field]?`. The definition not having attributes doesn't need its own message.

File: src/WeaverModelClass.coffee (hunk 2 above)
Comment (bastbijl) [reply src/WeaverModelClass.coffee]: its not, its for this case:
```Monkey:
  attributes:
    numberOfToes:
    numberOfFingers:
      key: ba```
now Monkey.attributes['numberOfToes']? is `false`

Comment (bastbijl) [reply src/WeaverModelClass.coffee]: but it should be true, because the attribute exists

File: src/WeaverModelClass.coffee (hunk 3 above)
Comment (bastbijl) [reply src/WeaverModelClass.coffee]: see previous comment
